templates/gov/index.html

```diff
@@ -342,81 +341,58 @@ <h2 class="p-muted-heading">Hardware partners</h2>
 
 <section class="p-strip">
   <div class="u-fixed-width">
-    <h2 class="u-sv2" id="certification-specs">Ubuntu Advantage, Ubuntu Pro, Ubuntu Pro FIPS specs</h2>
+    <h2 class="u-sv2" id="certification-specs">Ubuntu Pro specs</h2>
     <table role="presentation">
       <tr>
         <td><strong>FIPS 140-2 certified</strong></td>
         <td>
-          <div class="icon-container__icon">
-            <i class="p-icon--dark-tick"></i>
-          </div>
           <div class="icon-container__text">
-            FIPS 140-2 Level 1 certification for cryptographic modules on Ubuntu 16.04 LTS and Ubuntu 18.04 LTS.
+            FIPS 140-2 Level 1 certification for cryptographic modules on Ubuntu 16.04 LTS and Ubuntu 18.04 LTS. FIPS 140-3 modules for Ubuntu 22.04 LTS have been submitted to NIST and are now available for preview.
           </div>
         </td>
       </tr>
       <tr>
         <td><strong>DISA STIG</strong></td>
         <td>
-          <div class="icon-container__icon">
-            <i class="p-icon--dark-tick"></i>
-          </div>
           <div class="icon-container__text">
-            Defense Information System Agency (DISA) Security Technical Implementation Guide (STIG) for Ubuntu 16.04 LTS and Ubuntu 18.04 LTS.
+            Defense Information System Agency (DISA) Security Technical Implementation Guide (STIG) for Ubuntu 16.04 LTS and Ubuntu 18.04 LTS, and for Ubuntu 20.04 LTS with the Ubuntu Security Guide.
           </div>
         </td>
       </tr>
       <tr>
         <td><strong>Common Criteria EAL2 certified</strong></td>
         <td>
-          <div class="icon-container__icon">
-            <i class="p-icon--placeholder"></i>
-          </div>
           <div class="icon-container__text">
-            Common Criteria (CC) EAL2 certification for Ubuntu 16.04 Final certification from CSEC is pending for Ubuntu 18.04.
+            Common Criteria (CC) EAL2 certification for Ubuntu 16.04 Final certification from CSEC is pending for Ubuntu 18.04 and Ubuntu 16.04 LTS.
           </div>
         </td>
       </tr>
       <tr>
         <td><strong>CIS Benchmarks</strong></td>
         <td>
-          <div class="icon-container__icon">
-            <i class="p-icon--placeholder"></i>
-          </div>
           <div class="icon-container__text">
-            CIS has published benchmarks for Ubuntu 12.04 LTS and Ubuntu 14.04 LTS, with Ubuntu 16.04 LTS and Ubuntu 18.04 LTS benchmarks drafted.
-          </div>
+            CIS has published benchmarks for Ubuntu since 12.04 LTS, and Canonical has actively participated in drafting Ubuntu benchmarks for 22.04 LTS, 20.04 LTS, 18.04 LTS and 16.04 LTS releases.          </div>
```

Comment:
```suggestion
            CIS has published benchmarks for Ubuntu since 12.04 LTS, and Canonical has actively participated in drafting Ubuntu benchmarks for 22.04 LTS, 20.04 LTS, 18.04 LTS and 16.04 LTS releases.</div>
```